herbert_core/admin/is_idaaas.m

```diff
@@ -36,18 +36,14 @@
         size_suffix = '';
         return;
     end
-    
+
     cpu_pos = strfind(mess,'CPU(s)');
     mess = strsplit(mess(cpu_pos(1):end));
     n_cpu = str2double(mess{2});
     if n_cpu <10
```

Comment:
Why is large if `n_cpu < 10`? not `n_cpu > 10`?

Reply:
That is not what this statement says. This says `if n_cpu < 10; size_suffix = idaas_large`